pkg/plugins/cluster_test.go

```diff
@@ -32,69 +30,70 @@ var _ = Describe("DefinitionFiles", func() {
 		},
 		CrdName: "routes.test",
 	}
-	deployment := types.Capability{
-		Name:    "deployment",
-		Type:    types.TypeWorkload,
-		CrdName: "deployments.testapps",
-		Parameters: []types.Parameter{
-			{
-				Name:     "name",
-				Required: true,
-				Type:     cue.StringKind,
-				Default:  "",
-			},
-			{
-				Type: cue.ListKind,
-				Name: "env",
-			},
-			{
-				Name:     "image",
-				Type:     cue.StringKind,
-				Default:  "",
-				Short:    "i",
-				Required: true,
-				Usage:    "specify app image",
-			},
-			{
-				Name:    "port",
-				Type:    cue.IntKind,
-				Short:   "p",
-				Default: int64(8080),
-				Usage:   "specify port for container",
+	/*
+		deployment := types.Capability{
```

Comment:
this test is useful, please keep it in this PR.

Reply:
this doesn't work as it's testing against a github remote cue file that is not updated